test/functional/feature_block.py

```diff
@@ -828,7 +828,7 @@ def run_test(self):
         tx.vin.append(CTxIn(COutPoint(b64a.vtx[1].sha256, 0)))
         b64a = self.update_block("64a", [tx])
         assert_equal(len(b64a.serialize()), MAX_BLOCK_SIZE + 8)
-        self.send_blocks([b64a], success=False, reject_code=1, reject_reason=b'error parsing message')
+        self.send_blocks([b64a], success=False, reject_reason='non-canonical ReadCompactSize(): iostream error')
```

Comment:
14199: this breaks `feature_block.py` on mac, backporting 14007 should fix it

Reply:
actually just partially merged 14007 lol